examples/apps/hello_world/index.js

```diff
@@ -1,10 +1,14 @@
 var alexa = require('alexa-app');
+var pkg = require('./package.json');
+var applicationId = (typeof pkg.alexa == 'object' ? pkg.alexa.applicationId : "");
```

Comment:
What happens if someone forgets to put the applicationId in the package.json? At least a condition check is necessary for the alexa object if it exists. (I don't know if for an alexa-app it is mandatory/optional to put the applicationId in the package.json either.)